tests/writeResult/writeresult-getdeletedcount-002.phpt

```diff
@@ -24,6 +24,7 @@ var_dump($result->getDeletedCount());
 ?>
 ===DONE===
 <?php exit(0); ?>
---EXPECT--
+--EXPECTF--
+Deprecated: MongoDB\Driver\WriteResult::getDeletedCount(): Calling MongoDB\Driver\WriteResult::getDeletedCount() for an unacknowledged write is deprecated and will throw an exception in %s
```

Comment:
This uses `%s` prematurely but still passes. Contrast with the `getInsertedCount()` test.

Reply:
Find/replace was a little too eager here. Thanks!